locale/defaultMessages.json

```diff
@@ -3567,6 +3567,9 @@
     "context": "product price",
     "string": "Price"
   },
+  "src_dot_products_dot_components_dot_ProductList_dot_150865454": {
+    "string": "Simple"
```

Comment:
Adjectives should always have context provided - eg in polish, would you translate it as "prosty", "proste" or "prosta"?

Reply:
sure, I didn't know about it